bin/build-lib.js

```diff
@@ -116,7 +116,8 @@ async function buildLib () {
     const isTSX = file.endsWith('.tsx')
     if (isTSX || file.endsWith('.ts')) { plugins.push(['@babel/plugin-transform-typescript', { disallowAmbiguousJSXLike: true, isTSX, jsxPragma: 'h' }]) }
 
-    const { code, map } = await babel.transformFileAsync(file, { sourceMaps: true, plugins })
+    // compact and no comments because https://github.com/transloadit/uppy/pull/4868#issuecomment-1897717779
+    const { code, map } = await babel.transformFileAsync(file, { sourceMaps: true, plugins, comments: false, compact: true })
```

Comment:
I suggest we use a special env variable that we add to the GHA workflow to not impact our releases.
```suggestion
    const { code, map } = await babel.transformFileAsync(file, {
      sourceMaps: true,
      plugins,
      // compact and no comments because https://github.com/transloadit/uppy/pull/4868#issuecomment-1897717779
      comments: !process.env.DIFF_BUILDER,
      compact:  !!process.env.DIFF_BUILDER,
    })
```

Reply:
Also, `compact` puts all the code on one line, which is going to make the diff completely useless.

Reply:
And we can add `--ignore-blank-lines` to the `git diff` call in the workflow to minimize noise in the diff further

Reply:
good idea. but isn't it nice to remove comments from our releases too? then our bundle size will probably be significantly smaller

Reply:
I'd be surprised if comments are counted in the bundle size, that would create a bad incentive to not comment the code.

Reply:
Not sure I understand. The browser doesn’t count bundle size, it simply downloads the bundle. If there are comments in the bundle, would!’t the browser have to download a larger byte size versus a bundle with no comments?

Reply:
Browser would typically download `uppy.min.mjs`, which does not contain any comment.